app/front/scripts/directives/visualizations/piechart/template.html

```diff
@@ -8,6 +8,7 @@
     state="state"
     format-value="formatValue"
     messages="messages"
-    endpoint="{{ params.babbageApiUrl }}">
+    endpoint="{{ params.babbageApiUrl }}"
+    max-slices="5">
```

Comment:
Sure, this PR is blocked on https://github.com/openspending/babbage.ui/pull/62 then.